.travis.yml

```diff
@@ -19,7 +19,7 @@ matrix:
         - sbt -jvm-opts .travis-jvmopts ++$TRAVIS_SCALA_VERSION scalafmt::test test:compile test:doc
         - travis_wait 30 sleep 1800 &
         - sbt -jvm-opts .travis-jvmopts ++$TRAVIS_SCALA_VERSION test
-    - scala: 2.12.4
+    - scala: 2.12.6
```

Comment:
Is this at all worrisome? It's probably not necessary to keep; I think the travis cache was the issue. But if it's beneficial to bump the 2.12 version here I will of course leave it in.

Reply:
No not at all. Please leave it in. 

It’s a bit late here so I’ll do a more thorough review before merging tmw but so far looking great :)